powerquery-docs/data-privacy-firewall.md

```diff
@@ -10,7 +10,7 @@ ms.author: ehvonleh
 # Behind the scenes of the Data Privacy Firewall
 
 >[!NOTE]
->Privacy levels are currently unavailable in Power Platform dataflows. The product team is working towards re-enabling this functionality in the coming weeks.
+>Privacy levels are currently unavailable in Power Platform dataflows. The product team is working towards enabling this functionality in the future.
```

Comment:
```suggestion
>Privacy levels are currently unavailable in Power Platform dataflows, but the product team is working towards enabling this functionality.
```
upon re-evaluation, the specific future promise is not permitted in any case. proceeding with merging this (after a slight edit).